lib/method-map.js

```diff
@@ -0,0 +1,106 @@
+  '/session/:sessionId/element/:elementId/location_in_view': { GET: { command: 'getLocationInView' } },
+  '/session/:sessionId/element/:elementId/size': { GET: { command: 'getSize' } },
+  '/session/:sessionId/moveto': {
+    POST: {
+      command: 'moveTo',
+      payloadParams: { optional: ['element', 'xoffset', 'yoffset'] }
+    }
+  },
+  '/session/:sessionId/touch/click': {
+    POST: { command: 'click', payloadParams: { required: ['element'] } }
+  },
+  '/session/:sessionId/touch/perform': {
+    POST: {
+      command: 'performTouch',
+      payloadParams: { wrap: 'actions', required: ['actions'] }
+    }
+  },
+  '/session/:sessionId/touch/multi/perform': {
+    POST: {
+      command: 'performMultiAction',
+      payloadParams: { required: ['actions'], optional: ['elementId'] }
+    }
+  },
+  '/session/:sessionId/appium/device/shake': { POST: { command: 'mobileShake' } },
+  '/session/:sessionId/appium/device/lock': {
+    POST: { command: 'lock', payloadParams: { optional: ['seconds'] } }
+  },
+  '/session/:sessionId/appium/device/unlock': { POST: { command: 'unlock' } },
+  '/session/:sessionId/appium/device/is_locked': { POST: { command: 'isLocked' } },
+  '/session/:sessionId/appium/start_recording_screen': {
+    POST: {
+      command: 'startRecordingScreen',
+      payloadParams: { optional: ['options'] }
+    }
+  },
+  '/session/:sessionId/appium/stop_recording_screen': {
+    POST: {
+      command: 'stopRecordingScreen',
+      payloadParams: { optional: ['options'] }
+    }
+  },
+  '/session/:sessionId/appium/device/app_state': {
+    POST: {
+      command: 'queryAppState',
+      payloadParams: { required: [['appId'], ['bundleId']] }
+    }
+  },
+  '/session/:sessionId/appium/simulator/touch_id': {
+    POST: { command: 'touchId', payloadParams: { required: ['match'] } }
+  },
+  '/session/:sessionId/appium/simulator/toggle_touch_id_enrollment': {
+    POST: {
+      command: 'toggleEnrollTouchId',
+      payloadParams: { optional: ['enabled'] }
+    }
+  },
+  '/session/:sessionId/appium/app/launch': { POST: { command: 'launchApp' } },
+  '/session/:sessionId/appium/app/close': { POST: { command: 'closeApp' } },
+  '/session/:sessionId/appium/app/reset': { POST: { command: 'reset' } },
+  '/session/:sessionId/appium/app/background': {
+    POST: {
+      command: 'background',
+      payloadParams: { required: ['seconds'] }
+    }
+  },
+  '/session/:sessionId/appium/app/strings': {
+    POST: {
+      command: 'getStrings',
+      payloadParams: { optional: ['language', 'stringFile'] }
+    }
+  },
+  '/session/:sessionId/appium/element/:elementId/value': {
+    POST: {
+      command: 'setValueImmediate',
+      payloadParams: { required: ['text'] }
+    }
+  },
+  '/session/:sessionId/appium/receive_async_response': {
+    POST: {
+      command: 'receiveAsyncResponse',
+      payloadParams: { required: ['response'] }
+    }
+  },
+  '/session/:sessionId/appium/device/get_clipboard': {
```

Comment:
how about setClipboard?

Reply:
yep good catch, it wasn't generated because the deprecation flag was in the wrong place in appium. see https://github.com/appium/appium/pull/18050